index.html

```diff
@@ -25,10 +26,11 @@ <h3 class="cont">Contact</h3>
 
             </div>
             <ul>
-                <a class="btn"><i class="fa-solid fa-bars menu-bar"></i></a></button>
+                <button type="button" id="btn" onclick="reveal()" class="fa-solid fa-bars menu-bar">click</button>
             </ul>
         </nav>
     </header>
```

Comment:
### Your Menu
![mobile-menu-design (copy)](https://user-images.githubusercontent.com/93039359/196402445-5e4cc82f-693b-40f3-87a7-0d1c68bec24e.png)

### Figma Menu
![image](https://user-images.githubusercontent.com/93039359/196402110-65cc7b35-c6ec-4390-b92a-e1ffe2647e05.png)

- [ ] You can see in the image above that your menu is not matching with the actual design of the menu of your template. I have wrapped your design with the purple box :+1:
- [ ] You can see in the green boxes that your links and close button. These are not working. Kindly fix them :+1:
- [ ] You can see in the red box that your have a burger icon which has a keyword **CLICK**. Kindly remove that keyword :+1: to match the Figma template :blush: